components/EmotionForm.js

```diff
@@ -0,0 +1,189 @@
+import { emotions } from "@/lib/emotions";
+import { useState } from "react";
+import styled from "styled-components";
+
+export default function EmotionForm({ onCreateEmotion }) {
+  const currentDateTime = new Date(
+    new Date().getTime() - new Date().getTimezoneOffset() * 60000
+  )
+    .toISOString()
+    .slice(0, 16);
+
+  const [selectedEmotion, setSelectedEmotion] = useState("");
+  const [selectedIntensity, setSelectedIntensity] = useState(5);
+  const [selectedDateTime, setSelectedDateTime] = useState(currentDateTime);
+  const [notes, setNotes] = useState("");
```

Comment:
You don't need a separate state for your notes, neither do you need an onChange event.

Instead, you can extract the information of the entered note within your handleSubmit.

Either with form data or from the event.target.

What you did was create a controlled input component and you only need to control a component if you need to be informed about every change within the input and might want to display that information outside of that element.
(A great example is the selectedEmotion, where you display a text with the selectedEmotion value below the input element. This is where you need a controlled input element)